README.md

```diff
@@ -45,9 +45,13 @@ II_ENV=development dfx deploy --no-wallet --argument '(null)'
 Then the canister can be used as
 
 ```bash
-dfx canister call internet_identity register '(123, "test", vec {1; 2; 3}, null)'
+$ dfx canister call internet_identity init_salt
```

Comment:
The old call was probably using an old API. `init_salt` is the closest thing I could find to a health check, where the API is unlikely to break.